src/pages/home/ReportScreen.js

```diff
@@ -166,16 +161,24 @@ function ReportScreen({
     const {addWorkspaceRoomOrChatPendingAction, addWorkspaceRoomOrChatErrors} = ReportUtils.getReportOfflinePendingActionAndErrors(report);
     const screenWrapperStyle = [styles.appContent, styles.flex1, {marginTop: viewportOffsetTop}];
 
+    const isEmptyChat = useMemo(() => _.isEmpty(reportActions), [reportActions]);
     // There are no reportActions at all to display and we are still in the process of loading the next set of actions.
-    const isLoadingInitialReportActions = _.isEmpty(reportActions) && reportMetadata.isLoadingInitialReportActions;
+    const isLoadingInitialReportActions = isEmptyChat && reportMetadata.isLoadingReportActions;
```

Comment:
This `isLoadingInitialReportActions` is already present in main and I only changed the redundant `_.isEmpty` condition and moved it to a separate variable with memo `isEmptyChat`. I haven't introduced this `isLoadingInitialReportActions ` boolean and is something that main already contans.

Reply:
Oh that... it's resolved now, I think it might be a typo from me but it was fixed after pulling in the latest main and resolving the conflicts. Thanks for flagging !